src/java_bytecode/expr2java.cpp

```diff
@@ -124,7 +91,10 @@ std::string expr2javat::convert_code_function_call(
       unsigned p;
       std::string arg_str=convert(*it, p);
 
-      if(first) first=false; else dest+=", ";
+      if(first)
+        first=false;
+      else
+        dest+=", ";
       // TODO: ggf. Klammern je nach p
```

Comment:
I know it's not yours, @mgudemann, but maybe translate the comment to English ;)